### DISCUSSION TITLE
Add repo description to (email/twitter) release notifications

### DISCUSSION THREAD

**Discussion by jankatins** [community/community#37625] [Repositories]
I have subscribed to releases (Watch -> Custom -> check "Releases") to stay up to date with interesting repositories. Unfortunately, I sometimes forget what a repo slug actually means and so when I get a release notification, I have to open the repo to read up again on the project. It would be nice to have the repo description available in the email.

This would also be helpful in release pages generated for twitter, as I sometimes get retweeted releases (and issues/PR) and then do not know what the actual project is about.

**Comment by ossanna16** [comment]
Hi there @jankatins and welcome to our community! Thank you for the feedback 🙂
